lib/watir_drops/page_object.rb

```diff
@@ -97,8 +115,27 @@ def fill_form(model)
     def inspect
       '#<%s url=%s title=%s>' % [self.class, url.inspect, title.inspect]
     end
+
     alias selector_string inspect
 
+
+    def on_page?
```

Comment:
Hmmm, so we're thinking of having `#on_page?` wait for conditions to be met... I guess this makes sense.